aozora2html.gemspec

```diff
@@ -21,13 +21,18 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  if RUBY_VERSION < "2.4.0"
-    spec.add_dependency "rubyzip", "~> 1.3"
-  else
-    spec.add_dependency "rubyzip", "~> 2.0"
-  end
+  # Refer to current CI status at https://github.com/aozorahack/aozora2html/pull/39
+  spec.required_ruby_version = '>= 2.4', '< 3'
```

Comment:
I don't want to abandon Ruby < 2.4...

Reply:
Thank you for checking!

Let me ask two questions:

* What is the need to keep up with older Ruby versions? `aozora2html` is a command line tool. Ruby versioning tools like RVM and rbenv have become ubiquitous. Whether running on a server or used by users, it seems reasonable to install a relatively new Ruby processor separately even if older Ruby version has already been installed.
  * I am not sure about the following: the value of continuing to support older versions and the opportunity loss of code quality.
* If it is necessary to keep providing older Ruby versions, which specific versions and later does it support?

(I think maybe it's better to close this PR for now (and recreate another in the future). I'm sorry for my noise.)